docs/source/models.rst

```diff
@@ -1247,6 +1247,57 @@ models to be interpreted as generic Python functions for inference via
 You can also use the :py:func:`mlflow.prophet.load_model()`
 method to load MLflow Models with the ``prophet`` model flavor in native prophet format.
 
+Prophet pyfunc usage
+~~~~~~~~~~~~~~~~~~~~
+
+This example uses the `example_wp_log_peyton_manning.csv <https://github.com/facebook/prophet/blob/main/examples/example_wp_log_peyton_manning.csv>` _
+from Prophet's GitHub repository. A sample of the dataset is as follows:
+
+============= =================
+ds            y
+============= =================
+2007-12-10    9.59076113897809
+2007-12-11    8.51959031601596
+2007-12-12    8.18367658262066
+2007-12-13    8.07246736935477
+============= =================
+
+.. code-block:: py
+
+    import pandas as pd
+    from prophet import Prophet
+    import mlflow
+
+    train_df = pd.read_csv("https://raw.githubusercontent.com/facebook/prophet/main/examples/example_wp_log_peyton_manning.csv")
+    reg_model = Prophet(changepoint_prior_scale=0.5, uncertainty_samples=7)
```

Comment:
what does 'reg' stand for here?  Can we just name the variable `prophet_model` ?

Reply:
"reg" is short for "regression" ;) Anyway, renamed as suggested.